L/libcifpp/bundled/patches/mingw-no-ioctl.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/src/utilities.cpp b/src/utilities.cpp
+index 5238a5b..f5e54a8 100644
+--- a/src/utilities.cpp
++++ b/src/utilities.cpp
+@@ -43,7 +43,7 @@
+ #include <sstream>
+ #include <thread>
+ 
+-#if not defined(_MSC_VER)
++#if not (defined(_MSC_VER) || defined(__MINGW32__))
```

Comment:
More likely this should be `_WIN32`: https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170